src/components/profile/profileEdit/ProfileEdit.js

```diff
@@ -0,0 +1,23 @@
+const ProfileEdit = ({ closeEditModal }) => {
+  return (
+    <div className="edit-profile">
+      <div>
+        <span onClick={() => closeEditModal(false)}>X</span>
+        <h1>Edit Profile</h1>
+        <button>Save</button>
+      </div>
+
+      <div>
+        <textarea>name</textarea>
+
+        <textarea>Bio</textarea>
+
+        <textarea>Location</textarea>
+
+        <textarea>Website</textarea>
```

Comment:
These must definitely be for testing purposes

Reply:
yes I will still use form